src/main/java/com/example/user_management/model/response/CreateUserResponseModel.java

```diff
@@ -0,0 +1,19 @@
+package com.example.user_management.model.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@AllArgsConstructor
+public class CreateUserResponseModel {
+
+    @JsonProperty("id")
+    private Long id;
+
+    @JsonProperty("username")
+    private String username;
```

Comment:
you can use javax validation annotation like @NotNull or @NotEmpty to include validation on API level during serialization flow.

Reply:
Done